models/mpas_atm/model_mod.f90

```diff
@@ -4950,7 +5026,7 @@ subroutine convert_vert_distrib_state(state_handle, ens_size, location, quantity
    endif
 
    if (debug > 9 .and. do_output()) then
-      write(string2,'("zout_in_height:",F10.2)') zout
+      write(string2,'("zout[m]:",F10.2)') zout(1)
```

Comment:
Actually `zout[1]` ....

Reply:
I think the intent here is that the unit is meters.